tests/ast/data/col_literal.test

```diff
@@ -123,33 +123,33 @@ df = df.select(col("A") + float('-inf'))
 
 df = df.select(col("A") + float('nan'))
 
```

Comment:
just to verify: Is there `from decimal import Decimal` part of the ast runner setup?